sermant-backend/src/main/java/io/sermant/backend/handler/config/LoadbalancerPluginHandler.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sermant.backend.handler.config;
+
+import io.sermant.backend.entity.config.ConfigInfo;
+import io.sermant.backend.entity.config.PluginType;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Spring boot registry plugin handler
+ *
+ * @author zhp
+ * @since 2024-05-16
+ */
+public class LoadbalancerPluginHandler extends PluginConfigHandler {
+    private static final String TRAFFIC_MARKERS_CONFIGURATION_NAME_PREFIX = "servicecomb.matchGroup.";
+
+    private static final String LOADBALANCER_RULES_CONFIGURATION_NAME_PREFIX = "servicecomb.loadbalance.";
+
+    private static final String PATTERN = "^(app=[^&]*(&environment=[^&]*)?(&service=[^&]*)?)?$";
```

Comment:
this common pattern can be placed in PluginConfigHandler or common class

Reply:
pattern is not public，The pattern of each PluginHandler is different.